engine/src/test/java/com/datatorrent/stram/StreamingContainerManagerTest.java

```diff
@@ -1083,7 +1066,7 @@ public void testLatency() throws Exception
     GenericTestOperator o2 = dag.addOperator("o2", GenericTestOperator.class);
     HighLatencyTestOperator o3 = dag.addOperator("o3", HighLatencyTestOperator.class);
     GenericTestOperator o4 = dag.addOperator("o4", GenericTestOperator.class);
-    long latency = 5000; // 5 seconds
+    long latency = 2000; // 2 seconds
```

Comment:
What is the reason to go from 5 sec to 2 sec? Was it there to account for the first window?